src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/WorkloadManifest.json.in

```diff
@@ -84,16 +100,24 @@
       "abstract": true,
       "description": "tvOS Mono Runtime and AOT Workload",
       "packs": [
-        "Microsoft.NETCore.App.Runtime.Mono.tvos-arm64",
-        "Microsoft.NETCore.App.Runtime.Mono.tvossimulator-arm64",
-        "Microsoft.NETCore.App.Runtime.Mono.tvossimulator-x64",
         "Microsoft.NETCore.App.Runtime.AOT.Cross.tvos-arm64",
         "Microsoft.NETCore.App.Runtime.AOT.Cross.tvossimulator-arm64",
         "Microsoft.NETCore.App.Runtime.AOT.Cross.tvossimulator-x64"
       ],
-      "extends": [ "microsoft-net-runtime-mono-tooling" ],
+      "extends": [ "runtimes-tvos" ],
       "platforms": [ "osx-arm64", "osx-x64" ]
     },
+    "runtimes-tvos": {
+      "abstract": true,
+      "description": "tvOS Mono Runtime and AOT Workload",
```

Comment:
Why is this the only one that says "and AOT Workload"? it doesn't look different from ios or catalyst

Reply:
Missed shortening the description